### ISSUE TITLE
Disabling s3 devservice fails.

### ISSUE THREAD

**Issue by MarcusBiel** [quarkiverse/quarkus-amazon-services#675]
quarkus.s3.devservices.enabled=false

is ignored.

quarkus.devservices.enabled=false

works.

**Comment by overheadhunter** [comment]
Is this fix released yet? I can't prevent it from starting (and failing to do so), however I need other dev services, so I can't disable them globally.

**Comment by scrocquesel** [comment]
> Is this fix released yet? I can't prevent it from starting (and failing to do so), however I need other dev services, so I can't disable them globally.

@overheadhunter what is your configuration?

**Comment by overheadhunter** [comment]
@scrocquesel 

Well `quarkus.s3.devservices.enabled=false` (while `quarkus.keycloak.devservices` and `quarkus.datasource.devservices` are enabled).

Regardless, `localstack/localstack` attempts to start, which only works in rootful container, as apparently it wants to write to `/var/run/docker.sock` (I prefer podman in rootless mode).

```xml
<dependency>
  <groupId>io.quarkus.platform</groupId>
  <artifactId>quarkus-amazon-services-bom</artifactId>
  <version>3.20.0</version>
  <type>pom</type>
  <scope>import</scope>
</dependency>
...
<dependency>
  <groupId>io.quarkiverse.amazonservices</groupId>
  <artifactId>quarkus-amazon-s3</artifactId> <!-- resolves to version 3.3.1 -->
</dependency>
<dependency>
  <groupId>io.quarkiverse.amazonservices</groupId>
  <artifactId>quarkus-amazon-iam</artifactId> <!-- resolves to version 3.3.1 -->
</dependency>
```
